### PR TITLE
azure function to azure functions

### PR DESCRIPTION
**Pull request by v-prasadboke**
Required items, please complete
   
   Change(s):
   - Azure Function to Azure Functions

### CODE REVIEW

**Review by v-amolpatil — DISMISSED** [review]

**Review by v-sudkharat — DISMISSED** [review]
Looks good.

**Review by v-sudkharat — DISMISSED** [review]
Looks Good.

**Comment by v-sudkharat** [comment]
we are looking into this content, and we will provide next update on 07-07-2023 Thanks.

**Comment by v-sudkharat** [comment]
we have close this PR because description display issue in content hub will reopen this pr once fix that issue.

File: Solutions/AbnormalSecurity/Package/createUiDefinition.json [hunk 1]
```diff
@@ -60,7 +60,7 @@
             "name": "dataconnectors1-text",
             "type": "Microsoft.Common.TextBlock",
             "options": {
-              "text": "This solution installs the data connector for ingesting Abnormal Security Events logs into Microsoft Sentinel, using the Abnormal Security Events API. After installing the solution, configure and enable this data connector by following guidance in Manage solution view."
+              "text": "This Solution installs the data connector for AbnormalSecurity. You can get AbnormalSecurity custom log data in your Microsoft Sentinel workspace. After installing the solution, configure and enable this data connector by following guidance in Manage solution view."
```
Comment (v-sudkharat) [inline Solutions/AbnormalSecurity/Package/createUiDefinition.json:63]: @v-prasadboke Plz update data connector description here.

**Comment by v-sudkharat** [comment]
@v-prasadboke Need to add releasenotes for solution as per the guidance lines shared here https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/ReleaseNotesGuidance.md ,

**Review by v-sudkharat — DISMISSED** [review]
Looks Good.

**Review by v-sudkharat — APPROVED** [review]
We can see the required changes in solution after testing it. Thanks

**Pull request merged**